internal/check-imports/src/references.ts

```diff
@@ -36,7 +36,7 @@ const { log } = console;
 /**
  * abi-coder
  */
-log(AbiCoder);
+log(Interface);
```

Comment:
`Interface` is now the single entry point for interacting with a `JsonAbi`, not the `AbiCoder`.

Reply:
I was thinking of doing it inside of another PR. Per our private discussion, `AbiInterface` might not hit the mark because we've got `FunctionFragment`, and renaming `FunctionFragment` into `AbiFunction` or something similar feels kind of off, but `ProgramAbi` sounds good, as `ProgramAbiFunction` flows nicely.

Reply:
Sounds good 👍🏻